data/parse_smogon_stats.py

```diff
@@ -41,64 +38,29 @@ def get_smogon_stats_file_name(game_mode, month_delta=1):
 
 
 def get_pokemon_information(smogon_stats_url):
-    """Parses a Smogon stats document, such as: 'https://www.smogon.com/stats/2019-02/moveset/gen7ou-1825.txt'
-       Returns a dictionary containing the most likely spreads, items, and moves for each pokemon in order of likelihood
-    """
     r = requests.get(smogon_stats_url)
-    if r.status_code == 404:
-        r = requests.get(get_smogon_stats_file_name(ntpath.basename(smogon_stats_url.replace('-0.txt', '')), month_delta=2))
```

Comment:
This shouldn't be removed. This catches if the previous month's information does not yet exist and when it does: falls back to 2 months ago.

This is needed because the usage data typically isn't out until a few days into the month, so until about ~March 5th, the usage data from February won't exist, and the January data must be used